### DISCUSSION TITLE
上传多个文件会出错，关闭出错按钮之后已上传文件的文件有点奇怪

### DISCUSSION THREAD

**Discussion by IshuaWang** [halo-dev/halo#7158] [Q&A]
尝试上传六七个平均10M左右的文件，结果百分之百不成功。虽然前端显示不成功，但是有部分的文件还是被上传了，不过奇怪的一点是，相同的文件可能被上传好几次，后面重复的文件名是在原来的基础上又加了几个随机字符。不知道为什么会出现这种现象，为什么文件会被重复上传，还是说这些重复的并不是一次上传的结果？

**Comment by ruibaby** [comment]
没有复现上传失败的问题，如果你能稳定复现，请提供 Halo 的运行日志和浏览器的控制台日志。

> 相同的文件可能被上传好几次，后面重复的文件名是在原来的基础上又加了几个随机字符。不知道为什么会出现这种现象

不支持覆盖上传，同名文件会重命名。

**Reply by IshuaWang** [reply]
我下次传文件的记录一下，不过我还发现了另一个问题，就是一个mp3文件，如果文件名里带有";"，传上去之后就无法播放，不知道是为什么
